rsts/deployment/plugins/webapi/databricks.rst

```diff
@@ -381,3 +381,54 @@ Wait for the upgrade to complete. You can check the status of the deployment pod
   Make sure you enable `custom containers 
   <https://docs.databricks.com/administration-guide/clusters/container-services.html>`__
   on your Databricks cluster before you trigger the workflow.
+
+Databricks Agent Service configuration
+--------------------------------------
+
+You can use Spark via the Databricks Agent Service.
+
+To do so, follow these steps:
+
+1. Start the agent service using PyFlyte.
+2. Configure the Databricks Agent Service.
+3. Create a secret with the group "databricks" and the key "token."
```

Comment:
```suggestion
3. Create a secret with the group "databricks" and the key "api_token", and mount it to the Flyte agent deployment.
```

Reply:
Also, can we add a command here, so people will know how to create the secret? like
```bash
kubectl create secret ...
```

Reply:
Sure we can!